js_modules/dagster-ui/packages/ui-core/jest.config.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
     '\\.(css|less)$': 'identity-obj-proxy',
     '^worker-loader(.*)/workers/(.*)$': '<rootDir>/jest/mocks/$2',
     '^@dagster-io/ui-components$': '<rootDir>/../ui-components/src/index',
+    '^src/(.*)$': '<rootDir>/src/$1',
```

Comment:
What if we create an empty `shared` directory in the codebase, move the custom tsconfig.json file in there, and map shared to the appropriate src directory? 

We'd essentially require that you explicitly import .oss.tsx files from the "shared" path? Remapping a random+specific name like `shared` seems a bit less confusing than overriding `src`? Curious what you think

Reply:
I feel like `src` makes sense because the folder actually exists. But the concern about using an absolute path in some cases vs relative path in others being confusing is legit... so maybe making the import statement more special is justified... I'm not sure if autocomplete would work well if we changed it to a special keyword or something